src/entities/job.ts

```diff
@@ -25,48 +25,48 @@ export type Payload = {
   branchName: string;
   isFork: boolean;
   private: boolean;
-  isXlarge: boolean | null | undefined;
+  isXlarge?: boolean | null;
```

Comment:
It's probably out of scope for this, but we should really remove this at some point since we don't have different sizes of builders anymore and *everything* is Xlarge.

Reply:
I'll remove, it's actually a small change. I did a quick search, and we only ever set this value to true when we assign it a value at all.